spec/javascripts/live_search_spec.js

```diff
@@ -774,4 +774,46 @@ describe('liveSearch', function () {
       expect($('.js-selected-filter-count').html()).toBe('(6)<span class="govuk-visually-hidden"> filters currently selected</span>')
     })
   })
+
+  describe('GA4 tracking', function () {
+    beforeEach(function () {
+      window.GOVUK.getConsentCookie = function () {
+        return { settings: true }
+      }
+      window.GOVUK.analyticsGa4 = {}
+      window.GOVUK.analyticsGa4.Ga4FinderTracker = {}
+      window.GOVUK.analyticsGa4.Ga4FinderTracker.trackChangeEvent = function () {}
```

Comment:
I think you might have already fixed this in a later commit. Can you squash together related commits? Thanks.

Reply:
Apologies, I hadn't squashed anything as I wanted to be able to preserve the state before the `govuk_publishing_components` code was moved over, in case I was asked to move it back into `govuk_publishing_components`.